src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/SurveySvcClient.java

```diff
@@ -28,6 +100,126 @@
    * @return SurveyClassifierTypeDTO details of selector type requested.
    * @throws RestClientException something went wrong making http call.
    */
-  SurveyClassifierTypeDTO requestClassifierTypeSelector(UUID surveyId, UUID classifierType)
-      throws RestClientException;
+  @Retryable(
```

Comment:
The `@Retryable` annotation should be used very sparingly. We can't have users waiting 25 seconds for a response.